uploadNeuroDB/tarchiveLoader

```diff
@@ -243,7 +248,7 @@ if ($xlog) {
     $childPID = fork(); 
     if ($childPID == 0) { 
         exec("xterm -geometry 130x70 -e tail -f $logfile"); 
-        exit(0); 
+        exit $NeuroDB::ExitCodes::SUCCESS;
```

Comment:
@nicolasbrossard Could you check that I understood correctly this code and that it should indeed return success here?

Reply:
If `exec` runs the command successfully, it *never* returns. So whatever you put after `exec` will never be executed. You should remove the `exit....` line altogether. Also, you usually issue a call to `exec` like this:

`exec("my_command my_arg1 my_arg_2") or die "Command my_command my_arg1 my_arg2 failed: $!\n"`;